CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # Release Notes
 
+## Nerves v0.9.0
+
+* Update Notes
+
+Starting in Nerves v0.9.0 artifacts will no longer be fetched during
+`mix compile`. Artifact archives are intended to be fetched following `mix deps.get`.
```

Comment:
Can we add `now` to this sentence?  Otherwise it sounds like it may not actually be true yet.

`Artifact archives are now intended to be fetched following `mix deps.get`.`